objects/_o-arrow.scss

```diff
@@ -6,6 +6,18 @@
 /**
  * Creates arrow shapes using only CSS.
  *
+ * All the objects (available as silent placeholder selectors also):
```

Comment:
Can you change the wording as Objects include child elements (`__`) and modifiers (`--`) unlike utilities?